app/projects/components/joinProjectModal/index.tsx

```diff
@@ -21,12 +21,24 @@ export const JoinFields = z.object({
   hoursPerWeek: z.string().refine((val) => !val || /^\d+$/.test(val), {
     message: "Value must be an integer",
   }),
-  practicedSkills: z.array(z.any()),
+  practicedSkills: z.array(
+    z.object({
+      id: z.string(),
+      name: z.string().optional(),
+    })
+  ),
 })
 
 const JoinProjectModal = (props: IProps) => {
   const router = useRouter()
   const { createProjectMemberHandler } = useProjectMembers()
+
+  const INIT_VALUES_MODALFORM = useMemo(() => {
```

Comment:
@JuanGQCadavid was this useMemo necessary?

Reply:
I think you meant to tag @juanan0309

Reply:
It isn't absolutely necessary as the object is not complex and doesn't take too much time to re-compute. I did it that way because that is how it was implemented in the createForm 😅 , and it doesn't hurt anybody so I just implemented it the same way.